scripts/conversion_script.py

```diff
@@ -49,6 +49,59 @@ def __init__(self, path="", host="", keep=False):
         self.path = path
         self.host = host
         self.keep = keep
+        self.backup_suffix = ".backup"
+
+    def create(self, data=None):
+        if data is None:
+            # If no data to write supplied, download them from host
+            response = urlopen(self.host)
+            data = response.read()
```

Comment:
I think that we should split the the idea of creating from raw data and gathering data from an URL. This looks a bit confusing as we can pass a URL in the class constructor but also can pass data in this function. 

We could have an internal method like `_create` to encapsulate the directory/file creation and writting, but split the method to accept the host/data (probably in two different functions?)

My idea is to make this more readable and consistent, without making it too difficult to spot errors in case we need to do changes.

Reply:
:thinking: hmm alright, created separate methods for it, let me know if that is what you wanted